content/notero-item.ts

```diff
@@ -134,6 +135,26 @@ export default class NoteroItem {
     return Zotero.URI.getItemURI(this.zoteroItem);
   }
 
+  public getPDFURL(): string {
+    const zoteroAPIKey = getNoteroPref(NoteroPref.zoteroAPIKey);
+    const zoteroUserID = getNoteroPref(NoteroPref.zoteroUserID);
+    const attachmentIDs = this.zoteroItem
+      .getAttachments(false)
+      .slice()
+      // Sort to get largest ID first
+      .sort((a, b) => b - a);
+
+    for (const id of attachmentIDs) {
```

Comment:
In your [issue comment](https://github.com/dvanoni/notero/issues/4#issuecomment-1214555487), you mentioned that we could use `getBestAttachment()` for this. Just curious if that ended up not working for you?